src/index.ts

```diff
@@ -1,56 +1,30 @@
-import { Message } from './Message'
-import {
+export { Message, DecodedMessage } from './Message'
+export {
   PublicKey,
   PublicKeyBundle,
   SignedPublicKeyBundle,
   PrivateKey,
   PrivateKeyBundle,
 } from './crypto'
-import Stream from './Stream'
-import Client, {
+export { default as Stream } from './Stream'
+export {
+  default as Client,
   ClientOptions,
   ListMessagesOptions,
   SendOptions,
   Compression,
 } from './Client'
-import { Conversations, Conversation } from './conversations'
-import {
+export { Conversations, Conversation } from './conversations'
+export {
   ContentTypeId,
   ContentCodec,
   EncodedContent,
   ContentTypeFallback,
 } from './MessageContent'
-import { TextCodec, ContentTypeText } from './codecs/Text'
-import {
-  Composite,
-  CompositeCodec,
-  ContentTypeComposite,
-} from './codecs/Composite'
-import { SortDirection } from './ApiClient'
-
+export { TextCodec, ContentTypeText } from './codecs/Text'
 export {
-  Client,
```

Comment:
❤️ Really like seeing this trimmed down. What was the criteria you used to clean this up?